### PR TITLE
fix(Scrolling): removing clipping from Row and Column

### PR DESCRIPTION
**Pull request by slhay28**
## Description

adjusting Row and Column to account for changes made to clipping on renderer

## Changes

Removing clipping from Row and Column

## Testing

Check that Row and Column scroll correctly and do not have any components clipped out.

### CODE REVIEW

**Review by anthony9187 — APPROVED** [review]
🚀

File: src/packages/solid/components/Column/Column.tsx [hunk 1]
```diff
@@ -15,7 +15,6 @@ const Column: Component<ColumnProps> = (props: ColumnProps) => {
     <SolidColumn
       {...props}
       animate
```
Comment (chiefcll) [inline src/packages/solid/components/Column/Column.tsx:17]: remove animate everywhere too - doesn't do anything anymore

File: src/packages/solid/components/Row/Row.tsx [hunk 2]
```diff
@@ -15,7 +15,6 @@ const Row: Component<RowProps> = (props: RowProps) => {
       {...props}
       animate
       style={styles.Container}
-      clipping
       onSelectedChanged={withScrolling(props.x as number)}
```
Comment (chiefcll) [inline src/packages/solid/components/Row/Row.tsx:18]: Also need to support chaining onSelectedChanged so users can pass in another function

File: src/packages/solid/components/Column/Column.tsx [hunk 3]
```diff
@@ -15,9 +15,8 @@ const Column: Component<ColumnProps> = (props: ColumnProps) => {
...
-      clipping
       style={styles.Container}
-      onSelectedChanged={withScrolling(props.y as number)}
+      onSelectedChanged={props.OnSelectedChanged ?? withScrolling(props.y as number)}
```
Comment (chiefcll) [inline src/packages/solid/components/Column/Column.tsx:19]: typo - and you want both to be called ;-)

File: src/packages/solid/components/Row/Row.tsx [hunk 4]
```diff
@@ -10,13 +10,18 @@ export interface RowProps extends IntrinsicNodeProps {
 }
 
 const Row: Component<RowProps> = (props: RowProps) => {
+  const multiFunction = (callback: Function | any, originalFunc: Function) => {
```
Comment (chiefcll) [inline src/packages/solid/components/Row/Row.tsx:13]: lol - DRY - put this in utils and import it - we're going to need this everywhere

**Pull request merged**